docs/src/pages/components/slider/CustomizedSlider.js

```diff
@@ -48,104 +57,116 @@ const marks = [
   },
 ];
 
-const IOSSlider = withStyles({
-  root: {
+const iosComponents = {
```

Comment:
These overrides are basically just converted to the align with the new `components` prop, and each of them is adding styles on top of the already prepared styled slot components